spdx/file.go

```diff
@@ -105,9 +105,11 @@ type File2_2 struct {
 
 	// 4.4: File Checksum: may have keys for SHA1, SHA256 and/or MD5
 	// Cardinality: mandatory, one SHA1, others may be optionally provided
-	FileChecksumSHA1   string
-	FileChecksumSHA256 string
-	FileChecksumMD5    string
+	// FileChecksumSHA1   string
+	// FileChecksumSHA256 string
+	// FileChecksumMD5    string
+
+	FileChecksums []Checksum2_2
```

Comment:
Since each file cannot be associated with more than one checksums of same algorithm, it would be better to implement FileChecksums as  a map of ChecksumAlgorithm2_2 and Checksum2_2.

That would enable us to quickly get checksum for an algorithm. For example: if I want to access MD5 checksum value, we can do `fileObj.FileChecksum[MD5].Value` to get the value.